applications/io_board/targets/freertos.armv6m.st-stm32g0b1rc-railcom/main.cxx

```diff
@@ -0,0 +1,148 @@
+/** \copyright
+ * Copyright (c) 2013, Balazs Racz
```

Comment:
Should the copyright year here and below be updated?

Reply:
fixed